plot_builder.html

```diff
@@ -94,7 +94,7 @@
     var apiBase = "http://portaldev.sph.umich.edu/api/v1/";
     var data_sources = new LocusZoom.DataSources()
       .add("base", ["AssociationLZ", {url:apiBase + "single/", params: { analysis: 3 }}])
-      .add("ld", ["LDLZ" ,apiBase + "pair/LD/"])
+      .add("ld", ["LDLZ", "http://portaldev.sph.umich.edu/flask/v1/statistic/pair/LD/"])
```

Comment:
We probably shouldn't merge this particular URL into master. Other than that, this PR looks good.